decidim-meetings/app/services/decidim/meetings/calendar/component_calendar.rb

```diff
@@ -11,11 +11,9 @@ class ComponentCalendar < BaseCalendar
         #
         # Returns a String.
         def events
-          Rails.cache.fetch(cache_key) do
```

Comment:
@andreslucena Are you fine removing the cache from here? Or can you come up with another approach here?

The JS-approach you took for the back links wouldn't work here because we are not rendering HTML here. The idea here is to show only the events that match the current filters.

Reply:
Yes, I'm OK with that. Filters and caching aren't good friends.